manifests/config/apache.pp

```diff
@@ -212,10 +212,10 @@
   }
 
   $vhost_https_request_headers = [
-    'set X_FORWARDED_PROTO "https"',
-    'set SSL_CLIENT_S_DN "%{SSL_CLIENT_S_DN}s"',
-    'set SSL_CLIENT_CERT "%{SSL_CLIENT_CERT}s"',
-    'set SSL_CLIENT_VERIFY "%{SSL_CLIENT_VERIFY}s"',
+    'set X-FORWARDED-PROTO "https"',
```

Comment:
Will this break https://github.com/theforeman/puppet-pulpcore/blob/8288f3736a1543417eba6322e7013d0f230bd85e/templates/settings.py.erb#L65-L66?

Reply:
No, it will make it work!

`My-Cool-Header` gets translated to `HTTP_MY_COOL_HEADER` by gunicorn. It's just in the past it also allowed `My_Cool_Header` to become `HTTP_MY_COOL_HEADER` (you remember, huh?) and this used to work for us, and now it doesn't anymore.

Reply:
Also, puppet-pulpcore should really not depend on puppet-foreman setting things up -- it can be deployed standalone after all :)

Reply:
I see you came to the same conclusion.

Reply:
https://github.com/theforeman/puppet-pulpcore/pull/376